src/Atmos/Model/ref_state.jl

```diff
@@ -42,49 +36,61 @@ state.
 struct HydrostaticState{P, FT} <: ReferenceState
     virtual_temperature_profile::P
     relative_humidity::FT
+    subtract_off::Bool
```

Comment:
Is there a way to document what `subtract_off` is for in the docstring somehow? I can see how it's used, but will future people looking at this code be able to easily determine why this is here in the first place?

Reply:
Done